edc-tests/e2e-tests/src/test/java/org/eclipse/tractusx/edc/lifecycle/ParticipantRuntime.java

```diff
@@ -33,8 +33,14 @@ public class ParticipantRuntime extends EdcRuntimeExtension implements BeforeAll
     private DataWiper wiper;
 
     public ParticipantRuntime(String moduleName, String runtimeName, String bpn, Map<String, String> properties) {
+        this(moduleName, runtimeName, bpn, properties, true);
+    }
+
+    public ParticipantRuntime(String moduleName, String runtimeName, String bpn, Map<String, String> properties, boolean mockIdentity) {
         super(moduleName, runtimeName, properties);
-        this.registerServiceMock(IdentityService.class, new MockDapsService(bpn));
+        if (mockIdentity) {
```

Comment:
This could be replaced by a `!properties.contains("tx.ssi.miw.url")`, will save us from that unexpressive boolean